roles/openshift_web_console/tasks/install.yml

```diff
@@ -135,22 +135,22 @@
   when: not create_console_project.changed
 
 - name: Verify that the web console is running
-  command: >
-    curl -k https://webconsole.openshift-web-console.svc/healthz
-  args:
-    # Disables the following warning:
-    # Consider using get_url or uri module rather than running curl
-    warn: no
+  uri:
+    url: https://webconsole.openshift-web-console.svc/healthz
+    validate_certs: no
+    return_content: yes
+  environment:
+    no_proxy: '*'
   register: console_health
-  until: console_health.stdout == 'ok'
+  until: "'ok' in console_health.content"
   retries: 60
   delay: 10
   changed_when: false
   # Ignore errors so we can log troubleshooting info on failures.
   ignore_errors: yes
 
 # Log the result of `oc status`, `oc get pods`, `oc get events`, and `oc logs deployment/webconsole` for troubleshooting failures.
-- when: console_health.stdout != 'ok'
+- when: 'ok' in console_health.content
```

Comment:
Looks like we need quotes around this statement as well.

Reply:
Fixed, thanks